python/note_input_widget/ui/note_input_widget.py

```diff
@@ -1,17 +1,18 @@
 # -*- coding: utf-8 -*-
 
-# Form implementation generated from reading ui file 'note_input_widget.ui'
+# Form implementation generated from reading ui file 'C:\Users\qa\sg_envs\dev\tk\tk-framework-qtwidgets\python\note_input_widget\resources\note_input_widget.ui'
```

Comment:
That's not the first time I see that in a PR. Did something changed in our Qt res scripts that would force an absolute path? 

We are updating the script in shotgunsoftware/tk-core#977 so if there is a problem, we should address it there.

Reply:
I'm running on Windows, so I have my own powershell script to build resources...